packages/workers/merkle-worker/src/router.ts

```diff
@@ -1,35 +1,120 @@
 import { Router } from 'itty-router'
-import { createErrorResponse, ErrorCode } from './createResponse'
+import { createErrorResponse, createSuccessResponse, ErrorCode } from './createResponse'
 import { Env } from '.'
 import { WorkerRequest } from './utils'
 import { StandardMerkleTree } from '@openzeppelin/merkle-tree'
-import { Claim } from './types'
+import { Claim, MerkleData, MerkleTreeDump, MerkleProofResponse } from './types'
 
 const router = Router()
 
 router.post('/admin/api/merkle-root', async (request: WorkerRequest, env: Env) => {
```

Comment:
how are we authenticating admins on this admin route?

Reply:
With a separate encrypted bearer token that the public facing client app does not see.